app/src/main/res/values/dimens.xml

```diff
@@ -22,6 +22,8 @@
     <dimen name="menu_corner_radius">16dp</dimen>
     <dimen name="menu_horizontal_margin">4dp</dimen>
     <dimen name="menu_bottom_margin">12dp</dimen>
+    <dimen name="menu_item_height">48dp</dimen>
```

Comment:
Suggest to use a neutral term since it represents both item width and height ex. `menu_item_size`